setup.py

```diff
@@ -25,6 +26,15 @@ def grep(attrname):
 
 file_text = read(fpath('arrow/__init__.py'))
 
+install_requires = ['backports.functools_lru_cache',
+                    'python-dateutil',
+                    'simplejson']
```

Comment:
While Arrow is compatible with simplejson, it is not a requirement for installation.  Please move `'simplejson'` back to tox.ini.